modiscolite/meme_writer.py

```diff
@@ -0,0 +1,158 @@
+# Create a class that can be used to write MEME files.
+# Authors: Ivy Raine <ivy.ember.raine@gmail.com>, Jacob Schreiber <jmschreiber91@gmail.com>
+from typing import List, Optional
+import numpy as np
+
+from os import PathLike
+
+class MEMEWriterMotif:
+	"""Class for handling Motif for MEME file writing."""
```

Comment:
Aspirationally, we should have full docstrings on all our objects, with a <= 80 character header, a longer description, and a "Parameters" section with everything being passed into `__init__`. I won't hold up this specific PR over it, though.

Reply:
Done